doc/ResultsDiscussionWeaning.Rmd

```diff
@@ -0,0 +1,40 @@
+---
+title: "Results & Discussion (Weaning)"
+author: "Farida Samad-zada"
+date: '10 ������� 2017 � '
+output: html_document
+---
+##Results 
+According to the prediction of mixed-effects linear model, there is a negative relationship between the weaning age and the number of offspring across all orders (Fig. 1). Moreover, using orders as random effects significantly improves the model fit, compared to using generalized linear model, as indicated by the lower AIC value (Table 1). The overall linear trend across all orders appears to be negative for the weaning-number of offspring relationship. However, Rodentia, Insectivora and Primates show the most negative relationship with slopes being higher than 0.6 (Table 2). Additionally, despite all trends being negative there is order-specific variation in both the number of offspring, and weaning age (Fig. 1). Larger mammals tend to be clustered towards bottom right part of the plot, whereas small mammals are mostly concentrated in the upper-left corner (Figure 1). 
```

Comment:
Instead of describing  where they are in the plot, describe their attributes, e.g. "larger mammals tend to have .... while smaller animals.... and these two groups cluster separately when comparing the number of offspring vs weaning age". (if there are clear clusters, might be left out otherwise)

Choose either "Fig." or "Figure" and be consistent with the abbreviation throughout the document.